### ISSUE TITLE
[feature request] Workers support

### ISSUE THREAD

**Issue by CezaryDanielNowak** [jakiestfu/Blur.js#16]
When running blurjs, there is noticeable lag. Using Web Worker to transform image would fix the lag :)

**Comment by Huespal** [comment]
Yeah! You're right. Do you or someone have any clue on how to do that? I'm trying to make it possible, but the entire plugin is done in jQuery, and worker's need to be in native JavaScript.

**Comment by sailens** [comment]
If there is workers implementation I'd definitely use it. Lag is too heavy

**Comment by CezaryDanielNowak** [comment]
For now lag is caused also by events handled in wrong way. In my fork there is a lot of performance fixes:
https://github.com/korpirkor/Blur.js

**Comment by sailens** [comment]
I'll look into your fork, thanks @korpirkor !

**Comment by sailens** [comment]
The blur method you use is significantly faster, but also uglier. I'll study both to see which one suits me better, thanks!

**Comment by CezaryDanielNowak** [comment]
You can still use StackBlur instead of StackBoxBlur, just revert this commit https://github.com/korpirkor/Blur.js/commit/58878ab95bb0458b971760c8d007736348ca63b5 . I could make switch for blur algo but it would increase file size

**Comment by Huespal** [comment]
I've finally done it by using Web workers.
